pom.xml

```diff
@@ -121,63 +137,44 @@
     </scm>
 
     <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-checkstyle-plugin</artifactId>
+                    <version>3.0.0</version>
+                    <dependencies>
+                        <dependency>
+                            <groupId>com.puppycrawl.tools</groupId>
+                            <artifactId>checkstyle</artifactId>
+                            <version>8.17</version>
```

Comment:
GitHub recommends using version [8.18, ) as earlier version had 
![Screenshot from 2019-06-10 16-20-37](https://user-images.githubusercontent.com/23079344/59191543-7163c100-8b9d-11e9-838b-941f026c6e4d.png)
CVE.

Reply:
Automated PR has been generated, please review it. https://github.com/baymac/gitlab-branch-source-plugin/pull/12